### ISSUE TITLE
1.12.2 builds not detected by Forge

### ISSUE THREAD

**Issue by xJon** [Clintonio/NatureOverhaul#10]
From the log: `Disabling mod natureoverhaul it is server side only`.
I know this mod is not actively developed but I figured to have this here anyway.

**Comment by Clintonio** [comment]
Yeah my apologies but I have left the Minecraft modding community many years ago, I have left the source up in case anyone is interested in it though.
